apps/graphql-playground/src/components/ConfigScreen.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { Component, ChangeEvent } from "react";
+import { AppExtensionSDK } from "contentful-ui-extensions-sdk";
+import {
+  Card,
+  TextField,
+  Form,
+  TextLink,
+  Paragraph,
+  Note,
+  Tabs,
+  Tab,
+} from "@contentful/forma-36-react-components";
+import logoUrl from "../assets/logo.png";
+import sidebarScreenshotUrl from "../assets/sidebar.png";
+
+export interface AppInstallationParameters {
+  cpaToken: string;
+}
+
+type ParameterKeys = keyof AppInstallationParameters;
+
+interface ConfigProps {
+  sdk: AppExtensionSDK;
+}
+
+interface ConfigState {
+  currentTab: string;
+  parameters: AppInstallationParameters;
+}
+
+const TABS = [
+  {
+    id: "configuration",
+    label: "Configuration",
+    render: ({
+      sdk,
+      state,
+      onInputChange,
+    }: {
+      sdk: AppExtensionSDK;
+      state: ConfigState;
+      onInputChange: (event: ChangeEvent) => void;
+    }) => (
+      <>
+        <Paragraph>
+          The GraphQL Playground app enabled developers and content creators to
+          write GraphQL queries right next to their content.
+        </Paragraph>
+        <Form>
+          <Paragraph style={{ marginTop: "1em" }}>
+            <TextLink
+              href={`https://app.contentful.com/spaces/${sdk.ids.space}/api/keys`}
+              target="_blank"
+              rel="noopener"
+            >
+              Create a new pair of API keys
+            </TextLink>{" "}
+            and save the Content Preview API token below:
+          </Paragraph>
+          <TextField
+            name="cpaToken"
+            id="cpaToken"
+            labelText="CPA token"
+            required
+            value={state.parameters.cpaToken}
+            onChange={onInputChange}
+          />
+          <Note>
+            The CPA (Content Preview API) token allows you to also access
+            preview data when using GraphQL playground.
+          </Note>
+        </Form>
+      </>
+    ),
+  },
+  {
+    id: "sidebar-config",
+    label: "Sidebar",
+    render: () => (
+      <>
+        <Paragraph>
+          To enable GraphQL playground in the content entry sidebar head over to
+          the content model section and define to use a the custom sidebar. If
```

Comment:
Typo "define to use **a the** custom sidebar"

But main 🧠  phrase this properly at the moment 😆